actors/migration/power.go

```diff
@@ -40,12 +40,12 @@ func (m *powerMigrator) MigrateState(ctx context.Context, store cbor.IpldStore,
 		return nil, xerrors.Errorf("cron events: %w", err)
 	}
 
-	claimsRoot, err := m.updateClaims(ctx, store, inState.Claims, m.powerUpdates)
-	if err != nil {
+	// operates directly on inState
+	if err := m.updateClaims(ctx, store, inState.Claims, m.powerUpdates, &inState); err != nil {
```

Comment:
It's not redundant to pass `inState.Claims`.

Nit: put the `inState` parameter in this place rather than last.